quidem-ut/src/test/quidem/org.apache.druid.quidem.QTest/qaArray/ops_funcs_mv_funcs.03.msq.iq

```diff
@@ -9,10 +9,10 @@ FROM test_array;
 +-----+
 | col |
 +-----+
-|   1 |
-|   1 |
 |     |
 |     |
+| 1   |
+| 1   |
```

Comment:
This change, and the others in the file and in `qaArray/ops_funcs_mv_funcs.02.msq.iq`, look like improvements to me. The left-alignment of the values indicates they're now being treated as strings, which is more correct since these are `mv` functions.